scripts/generate.py

```diff
@@ -117,6 +117,24 @@ def generate_sample_help(subparsers, template):
         default=None,
     )
 
+def prompt_if_path_exists(output_dir, name):
+    path = os.path.join(output_dir, name)
+
+    if os.path.exists(path):
+        yes = {'yes', 'y'}
+        no = {'no', 'n', ''}
+        print(terminal_colors.WARNING
+              + f"The output directory {path} is not empty. Would you like to overwrite its content with the sample template? [y/N]"
```

Comment:
I assume the capitalized 'N' is meant to indicate that this is the default response if you just hit return, but that doesn't appear to be the behaviour of the script; it will print a message and wait for another input.

Reply:
Unfortunately, I was unable to reproduce the error. I tested it on:
Ubuntu 23.04,
Python 3.11.4,
zsh 5.9 (not sure if this one is important).
My colleague, whom I asked to test it on Windows, also couldn't reproduce this error. What system are you testing it on?

Reply:
Sorry - my bad. I completely missed the '' in the 'no' list which makes my review incorrect.